index.js

```diff
@@ -3241,6 +3264,83 @@
     impl: Z.size
   };
 
+  //# all :: Foldable f => (a -> Boolean) -> f a -> Boolean
+  //.
+  //. Returns `true` [iff][] all the elements of the structure satisfy the
+  //. predicate.
+  //.
+  //. See also [`any`](#any) and [`none`](#none).
+  //.
+  //. ```javascript
+  //. > S.all (S.odd) ([])
+  //. true
+  //.
+  //. > S.all (S.odd) ([1, 3, 5])
+  //. true
+  //.
+  //. > S.all (S.odd) ([1, 2, 3])
+  //. false
+  //. ```
+  _.all = {
+    consts: {f: [Z.Foldable]},
+    types: [$.Predicate (a), f (a), $.Boolean],
+    impl: curry2 (Z.all)
+  };
+
+  //# any :: Foldable f => (a -> Boolean) -> f a -> Boolean
+  //.
+  //. Returns `true` [iff][] any element of the structure satisfies the
+  //. predicate.
+  //.
+  //. See also [`all`](#all) and [`none`](#none).
+  //.
+  //. ```javascript
+  //. > S.any (S.odd) ([])
+  //. false
+  //.
+  //. > S.any (S.odd) ([2, 4, 6])
+  //. false
+  //.
+  //. > S.any (S.odd) ([1, 2, 3])
+  //. true
+  //. ```
+  _.any = {
+    consts: {f: [Z.Foldable]},
+    types: [$.Predicate (a), f (a), $.Boolean],
+    impl: curry2 (Z.any)
+  };
+
+  //# none :: Foldable f => (a -> Boolean) -> f a -> Boolean
+  //.
+  //. Returns `true` [iff][] none of the elements of the structure satisfies
+  //. the predicate.
+  //.
+  //. Properties:
+  //.
+  //.   - `forall p :: a -> Boolean, xs :: Foldable f => f a.
+  //.      S.none (p) (xs) = S.not (S.any (p) (xs))`
```

Comment:
Is this rendered correctly by markdown? I've never seen use of multiline `<code>` tags.

Reply:
It seems to work. See, for example, the [Markdown][1] for [`S.joinWith`][2].


[1]: https://github.com/sanctuary-js/sanctuary/blob/v0.15.1/index.js#L4639-L4644
[2]: https://sanctuary.js.org/#joinWith